### DISCUSSION TITLE
Critical: Add in the UI a way to make sure to use NVIDIA GPU please

### DISCUSSION THREAD

**Discussion by miguelgargallo** [open-webui/open-webui#5220] [General]
Critical: Add in the UI a way to make sure to use NVIDIA GPU please

**Comment by miguelgargallo** [comment]
https://github.com/open-webui/open-webui/discussions/348

**Reply by silentoplayz** [reply]
Hey @miguelgargallo, just to clarify: Open Web UI is not in control of Ollama's GPU usage - that's handled separately by Ollama itself (or perhaps it could be handled inside of Open WebUI with the `num_gpu (Ollama)` advanced parameter). If you're not seeing your NVIDIA GPU being used, it might be worth checking in with the Ollama team or updating your config to make sure everything is set up correctly.
